libs/model/src/globalActivityCache.ts

```diff
@@ -10,44 +10,90 @@ export async function getActivityFeed(models: DB, id = 0) {
    */
 
   const filterByCommunityForUsers = id
-    ? 'JOIN "Addresses" a on a.community_id=t.community_id and a.user_id = ?'
+    ? 'A.community_id = T.community_id and'
     : '';
 
   const query = `
-    WITH ranked_thread_notifs AS (
-        SELECT t.id AS thread_id, t.max_notif_id
-        FROM "Threads" t
-        ${filterByCommunityForUsers}
+    WITH ranked_thread_notifications AS (
+      SELECT 
+      T.id AS thread_id,
+      T.updated_at as updated_at,
+      json_build_object(
+        'id', T.id,
+        'body', T.body,
```

Comment:
are we getting the full body? I though we decided to truncate bodies when querying lists

Reply:
I am not sure about the truncating decision, but atm yes, full body. 

The `body` can be removed in some responses for lists, as we use `plaintext` for the preview in cases of thread lists. I think removing it from response would be a good idea.